cumulus/parachains/runtimes/assets/test-utils/src/test_cases_over_bridge.rs

```diff
@@ -127,7 +128,8 @@ pub fn limited_reserve_transfer_assets_for_native_asset_works<
 			);
 
 			// drip ED to account
-			let alice_account_init_balance = existential_deposit + balance_to_transfer.into();
+			let alice_account_init_balance =
+				existential_deposit.saturating_mul(2.into()) + balance_to_transfer.into();
```

Comment:
I think it should work without double ED with this fix https://github.com/paritytech/polkadot-sdk/pull/2042